lib/BloqadeWaveforms/src/waveform.jl

```diff
@@ -452,3 +517,60 @@ function sinusoidal(; duration, amplitude = one(duration))
         return amplitude * sin(2π * t)
     end
 end
+
+"""
+    function (..)(first, last)
+
```

Comment:
this is nice! wasn't thinking to document this at all.

Reply:
Made it a goal on my end to prioritize documenting anything that gets exported 😄 

This was particularly tricky to document (trying to get the docstring to show) but I got some inspiration looking at the way you documented things in `YaoSubspaceArrayReg`!